int8 rpm 				= 0
int8 raw_rpm 			= 1
int8 current_in 		= 2
int8 voltage_in 		= 3
int8 duty_cycle 		= 4
int8 temperature_mos 	= 5
int8 amp_hours 			= 6
int8 amp_hours_charged 	= 7
int8 watt_hours 		= 8
int8 watt_hours_charged = 9
int8 tachometer 		= 10
int8 tachometer_abs 	= 11
int8 fault_code 		= 12
int8 setpoint 			= 13

int8 id
time timestamp
int8 data_type
float32 value
